source/services/odnoklassniki.js

```diff
@@ -20,7 +20,7 @@ const odnoklassniki = {
     popupHeight: 400,
 };
 
-set(window, 'ODKL.updateCount', (index, counter) => {
+set(global, 'ODKL.updateCount', (index, counter) => {
```

Comment:
> Does this bring any benefits? 

I think it's more convenient to have the same behaviour in any environment: browser and server. If we can write into window in browser (we don't need actual window, if you will, notice that it's used just as global namespace) so we can do the same on server (using global)